web/templates/settings/generate_integration_url_modal.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="new-style">
+    <div class="input-group">
+        <label for="integration-url-modal-integration-input">{{t "Integration"}}</label>
+        <input type="text" id="integration-url-modal-integration-input" class="modal_text_input integration-url-parameter" autocomplete="off"/>
+    </div>
+    <div class="input-group">
+        <label for="integration-url-modal-stream-select-widget">{{t "Stream to use for notifications (optional)"}}</label>
+        <div id="integration-url-modal-stream-select-widget" class="dropdown-widget-button" type="button">
+            <span id="integration-url-modal-selected-stream" class="integration-url-parameter" data-unique-id="{{default_stream_select_option.unique_id}}">{{default_stream_select_option.name}}</span>
+            <i class="fa fa-chevron-down"></i>
+        </div>
+        <p>{{t "If you do not specify a stream, the bot will send notifications via PMs to the creator of the bot."}}</p>
+    </div>
+    <div class="input-group override-topic-group control-label-disabled">
+        <label class="checkbox">
+            <input type="checkbox" id="integration-url-modal-override-topic" class="integration-url-parameter" disabled/>
```

Comment:
I would have loved to use short class names instead of IDs but we have a label associated with each input and hence I don't think we can removed these IDs altogether.

Reply:
Ah, yes. I forgot about the label `for` attributes. We can think of shorter IDs in that case.

Reply:
I can remove the `integration-url-modal` prefix from the IDs but then the IDs won't be that unique and won't convey that they are being used here. I am also not able to think of something shorter that might work.

Reply:
I think we can remove "modal" for now and name them as `integration-url-selected-stream`, etc. And for the integration input I think it would be sufficient to name it `integration-input`.

Reply:
Makes sense. I have pushed the changes.